README.md

```diff
@@ -293,6 +298,31 @@ The credentials must have a policy applied that
 | AWS_ECR_DISABLE_CACHE        | true          | Disables the local file auth cache if set to a non-empty value     |
 | AWS_ECR_CACHE_DIR            | ~/.ecr        | Specifies the local file auth cache directory location             |
 | AWS_ECR_IGNORE_CREDS_STORAGE | true          | Ignore calls to docker login or logout and pretend they succeeded  |
+| AWS_ECR_REGISTRY_CONFIG_PATH | ~/.ecr        | Specifies the local file config directory location                 |
+
+#### Configure an AWS profile per registry
+
+In some instances, like when using SSO, you may want to override the default profile or credential resolution per registry.
+This can be achieved by creating a `registryConfig.yaml` file in the `~/.ecr` directory. To override this directory set the
+`AWS_ECR_REGISTRY_CONFIG_PATH` environment variable. This file configures registries to use a specific `AWS_Profile` as 
+defined in your AWS config. If no entry is found for a given repository it will fallback to the default AWS credentials.
+The general format for the file is:
+
+```yaml
+registryConfigs:
+  <registry>:
+    profile: "<AWS_PROFILE_NAME>"
+```
+
+For example:
+
+```yaml
+registryConfigs:
+  123456789000.dkr.ecr.ap-southeast-2.amazonaws.com:
+    profile: "Profile1"
+  987654321000.dkr.ecr.ap-southeast-2.amazonaws.com:
```

Comment:
Do prefix wildcards work here? I usually have one profile per account ID, but it would be shared across all regions.